WPThemeReview/Tests/CoreFunctionality/ThemeTemplatesException/footer_widgets.inc

```diff
@@ -0,0 +1,7 @@
+<!-- Annotation has to be on the second line as errors are thrown on line 1 and errors on annotation lines are ignored. -->
```

Comment:
The above comment can be removed from all test files for this sniff. For this sniff, it's fine to have the annotation on line 1.

The "_errors are thrown on line 1_" comes from the FileName sniff, where that is true, but that is not the case for this sniff (errors are thrown on the line where the variable is declared).